### PR TITLE
Dark mode of Home page and experience page

### PR DESCRIPTION
**Pull request by Adyasha8105**
I have made the page suitable for both modes.
Also done minute changes :) to make the site perfect.

@anushbhatia @smaranjitghose Have a look.

### CODE REVIEW

File: experience.html [hunk 1]
```diff
@@ -78,7 +78,7 @@ <h2 style="font-size: 3em;font-weight: bolder;" class="mb-0">Experience</h2>
     </section>
 
     <h4 class="text-center heading1">Work Experience</h4>
-    <h6 class="text-muted text-center py-2 heading2">These are an amazing oppurtunity to boost up your tech stack.</h6>
+    <h6 class="text-center py-2 heading2">These are an amazing oppurtunity to boost up your tech stack.</h6>
```
Comment (smaranjitghose) [inline experience.html:81]: Remove this

File: experience.html [hunk 2]
```diff
@@ -159,39 +159,39 @@ <h6 class="text-muted text-center py-2 heading2">I have been contributing variou
       <!-- Dynamic volunteership Card -->
     </main>
 
-    <h4 class="text-center heading1" style="margin-top:6rem">Mentorship</h4>
-    <h6 class="text-muted text-center py-2 heading2">I have been mentoring various projects for various organizations thourugh the skills
+    <h4 class="text-center heading1">Mentorship</h4>
+    <h6 class="text-center py-2 heading2">I have been mentoring various projects for various organizations thourugh the skills
```
Comment (smaranjitghose) [inline experience.html:163]: Remove this

File: experience.html [hunk 3]
```diff
@@ -66,9 +66,9 @@
               src="./assets/images//experience-page/experience.svg" draggable="true"> </div>
           <div
             class="px-md-5 p-3 col-md-6 d-flex flex-column align-items-center justify-content-center animate__animated animate__fadeInUp animate__delay-1s">
-            <h2 style="font-size: 3em;font-weight: bolder;" class="mb-0">Experience</h2>
-            <p class="font-weight-bolder text-muted text-center animate__animated animate__fadeInUp animate__delay-2s"
-              style="font-size: 1.1em; letter-spacing: 2px; line-height: normal;">
+            <h2 style="font-size: 3em;font-weight: bolder; color: var(--text);" class="mb-0">Experience</h2>
```
Comment (smaranjitghose) [inline experience.html:69]: avoid inline css at all costs

File: assets/js/main.js [hunk 4]
```diff
@@ -27,7 +27,7 @@ function topFunction() {
 
 let header = $(`
 <nav class="black">
-<div class="nav-wrapper container">
+<div class="nav-wrapper" style="margin: 0 20px 0 20px">
```
Comment (smaranjitghose) [inline assets/js/main.js:30]: Comments please. Remember our code should be readable to others as well

**Review by smaranjitghose — CHANGES_REQUESTED** [review]

File: experience.html (hunk 1 above)
Comment (Adyasha8105) [reply experience.html:81]: Remove all the heading part??

**Comment by Adyasha8105** [comment]
@smaranjitghose Is it okay now?

**Review by shreyakapoor08 — COMMENTED** [review]
looks fine now

**Comment by smaranjitghose** [comment]
Fix these bugs please
exp_bug
exp_bug_2

**Pull request merged**